Functions/Administration/xASL_adm_DefineASLSequence.m

```diff
@@ -1,4 +1,4 @@
-function [x] = xASL_adm_DefineASLSequence(x)
+function [x] = xASL_adm_DefineASLSequence(x, bVerbose)
```

Comment:
Please add bVerbose to line 4, line 10. And if it is an optional parameter, it needs admin on line 23 as well.

Reply:
Sorry for missing this. Fixed in 9afb01be1e81c615a976a2ae525063d7e2e3c0c2

Reply:
Thanks - we still need to list it in the INPUT parameter list...

Reply:
Done in 40c014963eebcc2c8d6d01a817980edd83077a5d